doc/whats_new/v1.6.rst

```diff
@@ -13,6 +13,28 @@ Version 1.6
   For a short description of the main highlights of the release, please refer to
   :ref:`sphx_glr_auto_examples_release_highlights_plot_release_highlights_1_6_0.py`.
 
+
+.. DELETE WHEN 1.6.0 IS RELEASED
+.. Changelog management has changed in v1.6.0.
+.. To add your changelog, create a file under the right folder under
+.. `doc/whats_new/upcoming_changes` with file format `<PULL REQUEST>.<TYPE>.rst`, where
+.. `<PULL REQUEST>` is a pull request number, and `<TYPE>` is one of:
+.. * `major-feature`
+.. * `feature`
+.. * `efficiency`
+.. * `enhancement`
+.. * `fix`
+.. * `api`
+.. Example of a changelog file:
+.. - :class:`ensemble.ExtraTreesClassifier` and :class:`ensemble.ExtraTreesRegressor`
+..   now supports missing values in the data matrix `X`. Missing-values are
+..   handled by randomly moving all of the samples to the left, or right child
+..   node as the tree is traversed.
+..   By :user:`Adam Li <adam2392>`
+.. See `doc/whats_new/upcoming_changes/README.md` for full details.
+.. Note only pull requests that result in a change relevant to users require a
+.. changelog entry (e.g., changes to private functions do not require a changelog).
```

Comment:
https://github.com/scikit-learn/scikit-learn/pull/30135#issuecomment-2432042934 was thumbed up by @adrinjalali so using this as a suggestion. @lucyleeow feel free to tweak it as you wish:

```suggestion
..
  TODO DELETE WHEN 1.6.0 IS RELEASED
  Since October 2024, DO NOT add your changelog entry in this file.

  Instead, create a file named `<PR_number>.<type>.rst` in the relevant sub-folder in
  `doc/whats_new/upcoming_changes/`. For full details, see:
  https://github.com/scikit-learn/scikit-learn/blob/main/doc/whats_new/upcoming_changes/README.md
```

Reply:
> Since October 2024, DO NOT add your changelog entry in this file

This is perfect, tells the user exactly when the change happened and includes nothing they don't need to know, i don't know why I didn't say this :sweat_smile: 

Note the extra `..` is because multiline comments do not seem to be able to contain blank lines